firebase-auth/src/commonTest/kotlin/dev/gitlive/firebase/auth/auth.kt

```diff
@@ -5,12 +5,13 @@
 package dev.gitlive.firebase.auth
 
 import dev.gitlive.firebase.*
+import kotlinx.coroutines.test.TestResult
 import kotlin.random.Random
 import kotlin.test.*
 
 expect val emulatorHost: String
 expect val context: Any
-expect fun runTest(test: suspend () -> Unit)
+expect fun runTest(test: suspend () -> Unit): TestResult
```

Comment:
Since we are using `kotlinx-coroutines-test` the result of runTest must be immediately return-ed from each test in JS.